server/routes_src/index.ts

```diff
@@ -12,6 +12,11 @@ router.post("/register", async (req, res, next) => {
   res.status(200).send({success: true, password: password});
 });
 
+router.post("/listing", async (req, res, next) => {
+  const success = await db.createListing(req.body.submitter, req.body.item, req.body.description, req.body.quantity, req.body.unit as any, req.body.type as any);
+  res.status(200).send({success});
+});
+
```

Comment:
wrap in login()?

Reply:
good point. I think I'll just do a separate PR and do an auth middleware or something to make it easier